website/blog/2020-07-20-docs-update.md

```diff
@@ -0,0 +1,36 @@
+- **Updated Core Component and API docs** We held a [documentation drive](https://github.com/facebook/react-native-website/issues/1579) to update reference docs! Thanks to these and other participants we were able to fully update the docs and add Snack examples to all of them in time for 0.62: [Marta Dabrowka](https://twitter.com/martadabrowka), [Abraham Nnaji](https://twitter.com/nnajiabraham), [Ahmed Talaat El-Hager](https://twitter.com/ahmdtalat), [Mohamed Abdel Nasser Abdou](https://twitter.com/mohamedsgap), [Danilo Britto](https://twitter.com/danilobrinu), [Mitul Savani](https://twitter.com/mitulsavani), [Kaio Duarte](https://twitter.com/kaiodduarte), [Pablo Espinosa](https://twitter.com/espipj), [Jesse Katsumata](https://twitter.com/natural_clar), [I Gede Agastya Darma Laksana](https://twitter.com/gedeagas), [Sebastião Bruno Kiafuka Fernando](https://twitter.com/bruno_kiafuka), [Miguel Bolivar](https://twitter.com/Darking360), [Dani Akash](https://twitter.com/dani_akash_), [Luiz Celso de Faria Alves](https://twitter.com/_eucelso), and [Bartosz Kaszubowski](https://twitter.com/simek). With their contributions, these are the best and most up to date React Native docs yet!
+- **ReactNative.dev** After 5 years we finally moved to our own domain! **reactnative.dev** is easier to autocomplete from your browser bar and is easier to type out than our previous **github.io** address!
+- **Design and site architecture improvements** to surface and categorize more of our guides and make content in the API reference more readable. Kudos especially to [Bartosz Kaszubowski](https://twitter.com/simek) whose attention to detail and collaboration has made many of these changes possible quickly!
+- **Keep those PRs coming!** We are able to consistently keep our open PRs under 10 per week! Thank you for sending them!
+
+Thank you so much to everyone who participated in the interviews, the survey, and our documentation efforts! Your collaboration makes this possible.
+
+## What’s next?
+
+The global COVID-19 pandemic has impacted many community members’ jobs. We are responding with additional content including:
+
+- New and improved Native Modules guides
+- Introductory content for people coming in to React Native for the first time
+
+### You can help!
+
+There are many ways you can help us write even better docs!
+
+1. If you see a typo, run into an issue with a guide, or something otherwise isn’t quiet right, click that “Edit” button and submit a PR.
+2. [Participate in our survey](https://www.surveymonkey.co.uk/r/DDZWQDJ)—this helps us understand how you use React Native and its documentation
+3. Write for us! We’re working on a tutorial section as well as guides for topics like offline apps, navigation, accessibility, debugging, animations, internationalization, and performance. If you or someone you admire or know is a perfect fit for any of these, please [reach out to me](http://emailto:rnabors@fb.com/)!
```

Comment:
This link doesn't work, did you mean `mailto://rnabors@fb.com`?

Also, mail links don't work super well, would it be better to refer to your twitter or our React Native account twitter?